data/wanderer/wanderer missions.txt

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) 2024 by Quantumshark
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Wanderers: Mereti: The Plant 1"
+	landing
+	name "Plant to <planet>"
+	passengers 2
+	cargo "strange plant" 1
+	description "Bring this plant (which seems to be of some significance to the Mereti) and two Wanderer 'mentors' to <destination>, so that the Wanderers can try to work out how to make it grow."
```

Comment:
This wording was suggested by bene-dictator [here](https://github.com/endless-sky/endless-sky/pull/9979#discussion_r1557124128). Not sure which suggestion to go with.

Reply:
Well, the use of parentheses feels more informal and less professional than commas, in my opinion. 

I work with writing at the Public Ministry and do not use parentheses on any official documents, sans for referencing parts of an administrative process, like this: (Event 12, p.06)

Of course, this isn't official documentation (and as such i can do _this!_), so go with what you find prettier! Either phrasing works, both grammatically and in how legible it will be in-game.

Reply:
I could rewrite it to be clearer and without brackets, I think.

```suggestion
	description "This plant seems to be of some significance to the Mereti. Bring the plant and two Wanderer 'mentors' to <destination> so that the Wanderers can try to work out how to make it grow."
```

Reply:
And indeed you did! I like this the best.